Gopkg.toml

```diff
@@ -96,44 +96,48 @@ required = [
   name = "github.com/giantswarm/micrologger"
 
 [[constraint]]
-  branch = "master"
+  branch = "k8s-1.15"
   name = "github.com/giantswarm/operatorkit"
 
 [[constraint]]
-  branch = "master"
+  branch = "k8s-1.15"
   name = "github.com/giantswarm/tenantcluster"
 
 [[constraint]]
   name = "github.com/google/go-cmp"
   version = "0.2.0"
 
+[[override]]
+  version = "v1.5.2"
+  name = "github.com/russross/blackfriday"
+
 [[constraint]]
   name = "github.com/spf13/viper"
   version = "1.0.0"
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.0"
 
 [[constraint]]
   name = "k8s.io/apiextensions-apiserver"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.0"
 
 [[override]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.13.1"
+  version = "kubernetes-1.15.0"
 
 [[constraint]]
   name = "k8s.io/apiserver"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.0"
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.0"
 
 [[constraint]]
   name = "k8s.io/helm"
-  version = "=2.12.0"
+  version = "=2.14.0"
```

Comment:
Same issue as in https://github.com/giantswarm/node-operator/pull/88#pullrequestreview-253854468

Reply:
Then `dep` fails with
```
Solving failure: No versions of github.com/giantswarm/helmclient met constraints:
        k8s-1.15: Could not introduce github.com/giantswarm/helmclient@k8s-1.15, as it has a dependency on k8s.io/helm with constraint 2.14.0, which has no overlap with existing constraint 2.12.0 from (root)
```